core/src/main/java/cz/xtf/core/config/BackwardCompatibility.java

```diff
@@ -72,4 +73,12 @@ private static String adminPassword() {
 
 		return password1 != null ? password1 : password2;
 	}
+
+	private static String routeDomain() {
+		String configDomain = XTFConfig.get("xtf.config.domain");
+		String configRouteDomain = XTFConfig.get("xtf.config.route_domain");
+		String systemEnvDomain = System.getenv().get("DOMAIN");
+
+		return configDomain != null ? configDomain : (systemEnvDomain != null ? systemEnvDomain : configRouteDomain);
```

Comment:
The backwards compatibility interpretation of DOMAIN (or xtf.config.domain) should be

routeDomain = "apps".$DOMAIN

see https://github.com/xtf-cz/xtf/blob/master/utilities/src/main/java/cz/xtf/XTFConfiguration.java#L173

Reply:
@maschmid Right, great catch! Thanks, Marek. Should be fixed now.